drivers/hmis/spec/support/form_helpers.rb

```diff
@@ -35,317 +35,194 @@ def add_item_to_definition(definition, item)
     definition
   end
 
-  def completed_form_values_for_role(role)
-    yield(COMPLETE_VALUES[role]) if block_given?
+  # get mocked "hud_values" (keyed by field name) and generate "values" (keyed by link id)
+  def mock_form_values_for_definition(definition)
```

Comment:
These changes can be pushed totally separately, I'm just trying to do the most low-effort way to remove the dependency on link ids from this test. This seems to work.